Configure.pl

```diff
@@ -970,10 +969,6 @@ =head1 OPTIONS
 
 Disable JIT compiler, which is enabled by default to JIT-compile hot frames.
 
-=item --lua=path/to/lua/executable
-
-Path to a lua executable. (Used during the build when JIT is enabled).
-
```

Comment:
I'd really have to check the commit that introduced it, but  recall that different versions of lua have incompatibilities in how they read hexadecimal numbers on some operating systems (that may have been solaris), and that affects how offsets are read, etc. (This may have been solaris). Also, having a system lua requires installation of bit modules which are included in this minilua (or in luajit). So from my PoV:
- we kind of have to ship minilua anyway for correctnes
- it is more user-friendly to do it this way